examples/planning/train.ipynb

```diff
@@ -0,0 +1,332 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Planning for Self Driving Vehicles\n",
+    "\n",
+    "In this notebook you are going to train your own ML policy to fully control a SDV. You will train your model using the Lyft Prediction Dataset and [L5Kit](https://github.com/lyft/l5kit).\n",
+    "\n",
+    "**Before starting, please download the [Lyft L5 Prediction Dataset 2020](https://self-driving.lyft.com/level5/prediction/) and follow [the instructions](https://github.com/lyft/l5kit#download-the-datasets) to correctly organise it.**\n",
+    "\n",
+    "The policy will be deep neural network (DNN) which will be invoked by the SDV to obtain the next command to execute.\n",
+    "\n",
+    "More in details, you will be working with a CNN architecture based on ResNet50.\n",
+    "\n",
+    "![model](../../images/planning/model.svg)\n",
+    "\n",
+    "\n",
+    "#### Inputs\n",
+    "The network will receive a Bird's-Eye-View (BEV) of the scene surrounding the SDV as the only input. This has been rasterised in a fixed grid image to comply with the CNN input. L5Kit is shipped with various rasterisers. Each one of them captures different aspects of the scene (e.g. lanes or satellite view).\n",
+    "\n",
+    "This input representation is very similar to the one used in the [prediction competition](https://www.kaggle.com/c/lyft-motion-prediction-autonomous-vehicles/overview). Please refer to our [competition baseline notebook](../agent_motion_prediction/agent_motion_prediction.ipynb) and our [data format notebook](../visualisation/visualise_data.ipynb) if you want to learn more about it.\n",
+    "\n",
+    "#### Outputs\n",
+    "The network outputs the driving signals required to fully control the SDV. In particular, this is a trajectory of XY and yaw displacements which can be used to move and steer the vehicle.\n",
+    "\n",
+    "After enough training, your model will be able to drive an agent along a specific route. Among others, it will do lane-following while respecting traffic lights.\n",
+    "\n",
+    "Let's now focus on how to train this model on the available data.\n",
+    "\n",
+    "### Training using imitation learning\n",
+    "The model is trained using a technique called *imitation learning*. We feed examples of real driving experiences to the model and expect it to take the same actions as the driver did in those episodes. This is very similar to how tasks like classification are usually solved. This is also the same concept we use in our [motion prediction notebook](../agent_motion_prediction/agent_motion_prediction.ipynb), so feel free to check that out too.\n",
```

Comment:
(expert) driver maybe?

Reply:
Might consider replacing 'similar to how tasks like classification ...' by using supervised learning, maybe makes it sound a bit more formal?
E.g. something like - 'Imitation Learning is a subfield of supervised learning, in which one tries to learn a function f: X -> Y describing given input / output pairs - one prominent example of this is image classification.'